src/types.js

```diff
@@ -329,7 +333,7 @@ export const fromKebab = x => x.replace(/-/g, '_')
 
 function createJsonTransform(fn) {
   return function jsonTransform(x, column) {
-    return column.type === 114 || column.type === 3802
+    return (x && typeof x in allowList) && (column.type === 114 || column.type === 3802)
```

Comment:
Instead of this extra `allowList` variable, you can probably just do the check inline.

Also, you may want to instead try checking against:

- Is `x` an array
- Is `x` an object
- Is `x` not `null`

It's possible that you don't need 3 different conditions for this though...

Reply:
Commit
https://github.com/porsager/postgres/pull/510/commits/b0b2e0cca3f0ab0becc42d0cb689fdd1fa0e5f6e

Reply:
seems better to me 👍 

```js
return typeof x === 'object' && x !== null
```

@porsager condition good for you too?

Reply:
Yup - perfect!